msgs/odom.proto

```diff
@@ -0,0 +1,13 @@
+syntax = "proto2";
+package odom_msgs.msgs;
+
+message odom
+{
+  required int32 usec    = 1;
+  required float x       = 2;
+  required float y       = 3;
+  required float z       = 4;
+  required float roll    = 5;
+  required float pitch   = 6;
+  required float yaw     = 7;
```

Comment:
@ChristophTobler probably you can add some `optional` fields like:
1. vx, vy, vz
2. the variances for x|y|z, vx|vy|vz and r|p|y (not the covariances as that would require propagating a matrix)
since we are talking about an Odometry msg (parallel to what the odom msg on ROS offers).

Reply:
The above could be used later on an RTPS msg or directly on `LOCAL_POSITION_NED_COV`.